pattern-lab/source/_patterns/00-atoms/02-text/01-paragraph.twig

```diff
@@ -14,9 +14,9 @@
 {% if paragraph_content %}
 {% set paragraph_base_class = paragraph_base_class|default('paragraph') %}
 
-<p >
+<p>
   {% block paragraph_content %}
-    {{ paragraph_content }}
+    {{ paragraph_content|striptags }}
```

Comment:
no striptags, as paragraph can have 'a', 'strong' and other tags

Reply:
Removed the striptags